README.rst

```diff
@@ -129,8 +129,8 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.io/newcomer>`__
-to help you get started by fixing an issue yourself! If you get stuck anywhere
+`Newcomers Guide <http://coala.io/newcomer>`__ page
```

Comment:
I think there is no need for the word `page` here. `We also have a Newcomers Guide to...` is fine.